ports/stm32/mboot/main.c

```diff
@@ -77,6 +77,80 @@
 #endif
 #endif
 
+/******************************************************************************/
```

Comment:
Since main.c is getting large, how about moving all this to a file like `dfu.h`?